.env

```diff
@@ -0,0 +1,4 @@
+GEOCODE_API_KEY = d6769691832077
```

Comment:
Oops! You've got all your keys here for the world to see. Looks like you forgot to make a .gitigore file and add your .env to it.